rolo/routing/router.py

```diff
@@ -166,6 +166,31 @@ def add(
         """
         ...
 
+    @overload
+    def add(
+        self,
+        path: str,
+        endpoint: Callable[[Request, ...], Response],
+        host: t.Optional[str] = None,
+        methods: t.Optional[t.Iterable[str]] = None,
+        **kwargs,
+    ) -> Rule:
+        """
```

Comment:
I've copied the entire docstring from the previous `overload`, including the TODO - I only changed the description of the `endpoint`-argument. Happy to adjust this!